interface/src/app/gene/utils.js

```diff
@@ -14,6 +14,17 @@ angular.module('adage.gene.utils', [
     updatePageNumbers: function(scope) {
       scope.pageDict.page = scope.page;
       scope.updatePage(scope.page);
+    },
+
+    getGeneLabel: function(gene) {
+      // Returns standard_name if it exists and systematic_name otherwise.
+      var geneLabel = gene.systematic_name;
+
+      if (gene.standard_name) {
+        geneLabel = gene.standard_name;
+      } else {}
```

Comment:
Ah good catch, I guess not. For some reason I thought it was required but I see that it isn't, and this piece of code is probably simple enough that adding the empty ``else {}`` does not add any clarity. I will take it out.